buildkite/build_release.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -xueo pipefail
+trap 'echo build script failed at line $LINENO' ERR
+
+TODAY=$(date +'%Y-%m-%d')
+: ${BRANCH:=master}
+
+if [ ! -d installer ]; then
+    git clone https://github.com/dlang/installer.git installer
+fi
+pushd installer/create_dmd_release
+
+git reset --hard --quiet
+git clean -df --quiet
```

Comment:
You need a double `-ff` to remove cloned sub repositories (not sure whether you want to).
For the other jobs we do this to avoid any potential side effects from previous builds.